javacomparator/DocxFile.java

```diff
@@ -0,0 +1,109 @@
+import org.json.simple.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+
+/**
+ * DocxFile Class implments the methods required for comparasions of DOCX files
+ */
+public class DocxFile extends OoxmlFile {
+
+    public final String type = "docx";
```

Comment:
Constants should be denoted by CAPITALS (as mentioned already in V0 comments). Please change in all files - example here: TYPE